data/projects.json

```diff
@@ -149,5 +149,14 @@
         "site": "",
         "tags": "python-project,personal-assistant,python-assistant"
     }
-
+    {
+        "name": "exchangerate-javascript-sdk",
+        "display": "Currency Converter",
+        "description": "Unofficial JavaScript SDK for Exchange rates API",
+        "author": "reliut-g",
+        "category": "website",
+        "level": "basic",
+        "site": "https://github.com/reliut-g/exchangerate-javascript-sdk",
```

Comment:
Your site URL should be `https://reliut-g.github.io/exchangerate-javascript-sdk/`